autode/smiles/parser.py

```diff
@@ -537,6 +540,21 @@ def atomic_n_hydrogens(string):
     return 0
 
 
+def atomic_class(string) -> Optional[int]:
+    """Extract the atomic class from a string i.e.::
+
+        H4:2  -> 2
+        :7    -> 7
+        :001  -> 1
+    """
+
+    if ':' not in string:
+        return None
+
+    digits = string.split(':')[1]
+    return int(digits)
+
+
 def next_char(string, idx):
     """
     Get the next character in a string if it exists otherwise return
```

Comment:
Move object type into def to be consistent? E.g. string (str): -> string:

Reply:
going to do it for this function, but there's so many others 😢